aioredisqueue/queue.py

```diff
@@ -82,6 +97,9 @@ def _put_pipe(self, task_id, task_payload):
         )
 
     def put(self, task, method='lua'):
+        if self._is_stopped:
+            raise exceptions.Stopped
```

Comment:
This is an error: you rising an instance of type `type`. (Pylint should be able to find this)